src/rextester.js

```diff
@@ -74,12 +74,12 @@ async function outputResult(language, code, compilerArgs, callback) {
             };
         }
     }
-    return callback(requestResponse);
+    return requestResponse;
 }
 
 async function handleRequest(language, code, imports, callback) {
     let compilerArgs;
-
+    
     if (!code || !code[0] ) {
         return callback( {
```

Comment:
It seems that you forgot to remove callback() here.
(same thing few lines below.)